### PR TITLE
feat: _ image update_

### PR DESCRIPTION
**Pull request by adkumar1**
- This PR  contains image creation of image in docker hub

### CODE REVIEW

File: .github/workflows/dataexchange-pipeline.yml [hunk 1]
```diff
@@ -35,12 +35,9 @@ on:
...
   workflow_dispatch:
   
-
 env:
-  # Use GH Container Registry instead of docker
-  # github.repository equals to <account|org>/<repo>
-  REGISTRY: ghcr.io
-  IMAGE_NAME: catenax-ng/tx-data-exchange-test-service/dataexchange
+  IMAGE_NAMESPACE: "tractusx"
+  IMAGE_NAME: "dataexchange"
```
Comment (SebastianBezold) [inline .github/workflows/dataexchange-pipeline.yml]: This is a very generic name. Please think of a more distinct name

File: README.md
Comment (SebastianBezold) [inline README.md:1]: The "Notice for Docker images" is missing in this file. 
This has to be added as described in [TRG 4.06](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-06)

**Review by SebastianBezold — CHANGES_REQUESTED** [review]

File: README.md
Comment (adkumar1) [reply README.md:1]: Updated the notice & image name

**Comment by SebastianBezold** [comment]
Hi @adkumar1,

I think `dataex` is even worse than `dataexchange`. If there is no better description than a general "data-exchange", then i would suggest the long version `data-exchange-test-service` to indicate, that it is for testing.
Also, please update the workflows like trivy to use the images from DockerHub instead of pointing to GitHub packages is a fork

**Review by SebastianBezold — APPROVED** [review]

**Pull request merged**